spec/manageiq/base_spec.rb

```diff
@@ -85,9 +85,9 @@
     let(:logger) { described_class.new(@log) }
 
     it "truncates long messages when max_message_size is set" do
-      msg = "a" * 1_572_864 # 1.5 mb in bytes
+      msg = "a" * 10.kilobytes
```

Comment:
shouldn't this be 8.kilobytes ?

Reply:
I think he's trying to show a longer message gets truncated

Reply:
Ahh you're right (this is the spec :) )